packages/@guardian/source-foundations/src/typography/typography.stories.mdx

```diff
@@ -12,6 +12,9 @@ import {
 	ItalicsRenderer
 } from '../../../../../.storybook/typographyRenderers';
 
+<!--TODO: hide additional sidebar stories when functionality becomes available,
```

Comment:
not anything to change at all — but wouldn't it be great to have a gh bot / some kind of tool that reminded us about TODOs periodically? otherwise they're just kinda left until someone happens to come across them in the future :)